plugins/gitlab/api.ml

```diff
@@ -269,7 +269,7 @@ let head_commit t repo =
 
 module Commit = struct
   module Set_status = struct
-    let id ="gitlab-set-status"
```

Comment:
Rather than having this hardcoded, you should be able to override the value with something meaningful for the application. Eg. for ocaml-ci this makes sense but for mirage-ci or tezos-ci it should be different.